mathics/eval/inference.py

```diff
@@ -121,17 +121,8 @@ def remove_nots_when_unnecesary(pred, evaluation):
 
 
 def get_assumptions_list(evaluation):
-    assumptions = None
-    assumptions_def = evaluation.definitions.get_definition(
-        "System`$Assumptions", only_if_exists=True
-    )
-    if assumptions_def:
-        assumptions = assumptions_def.ownvalues
-        if len(assumptions) > 0:
-            assumptions = assumptions[0].replace
-    if assumptions is None:
-        return None
-
+    """Return a list of assumptions"""
```

Comment:
This is an example of how the change helps me to find places where the code can be made simpler. We can now discard the change in the attributes of `Definition` and keep this.

Reply:
It is more modular is is not quite the same as simpler.